vllm/v1/core/sched/scheduler.py

```diff
@@ -331,9 +351,10 @@ def schedule(self) -> SchedulerOutput:
                     else:
                         logger.debug(
                             "%s is still in WAITING_FOR_REMOTE_KVS state.",
-                            request.request_id)
-                        self.waiting.popleft()
-                        skipped_waiting_requests.appendleft(request)
+                            request.request_id,
+                        )
+                        skipped_waiting_requests.prepend_request(
+                            self.waiting.pop_request())
```

Comment:
Let's keep like b4 here
```suggestion
			self.waiting.pop_request()
                        skipped_waiting_requests.prepend_request(request)
```